### PR TITLE
LPS-79936

### PR DESCRIPTION
**Pull request by brianchandotcom**

### CODE REVIEW

**Comment by brianchandotcom** [comment]
@istvansajtos see https://github.com/brianchandotcom/liferay-portal/pull/149689

**Comment by istvansajtos** [comment]
Hi @brianchandotcom ,

For your [concern](https://github.com/brianchandotcom/liferay-portal/pull/149689#issuecomment-2080619947), I want to verify that AccessControlAdvice runs for JSON web services. For this, the services
1. must be AopService instances to be intercepted by advices (see testAopService method)
2. must be annotated with @AccessControlled so that AccessControlAdvice runs for them. (see testAccessControlled method)

Basically I'm testing only the design, but the goal is to verify that AccessControlAdvice will intercept the service calls. 

Based on this do you have any suggestion on how to name the test class?

**Comment by brianchandotcom** [comment]
Sure, I'm sure there is a class in the code base that fits the existing pattern. Go find it. @istvansajtos
